### PR TITLE
Feature/58 name and description page

### PR DESCRIPTION
**Pull request by Limbo2332**

### CODE REVIEW

**Comment by Limbo2332** [comment]
[Name and description page](https://trello.com/c/ar8TMOeM/58-name-and-description-page)

File: frontend/src/app/shared/components/editor-question/editor-question.component.sass [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+.editor
+  background-color: $strong-black
+  color: $white
+  min-height: 260px
+  padding: 10px
+
+// Markdown editor styles
+
+::ng-deep .md-header
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/editor-question/editor-question.component.sass:78]: please use ::host with ::ng-deep

File: frontend/src/app/shared/components/editor-question/editor-question.component.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    public previewMarkDown() {
+        this.selectedTab = TabType.Preview;
+        this.bsEditorInstance.showPreview();
+    }
+
+    public submitQuestion() {
+        this.selectedTab = TabType.Help;
+
+        // add logic with submit
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/editor-question/editor-question.component.ts:53]: please delete all comments

File: frontend/src/app/shared/components/editor-question/editor-question.component.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                <img class="category-icon" src="/assets/images/categories-icons/rubik-icon.png" alt="rubik-icon" />
+                <span class="category-text">Puzzles</span>
+            </div>
+        </div>
+    </div>
+    <div class="input-wrapper mt-12px d-flex flex-column gap-2">
+        <p class="question-header">Name *</p>
+        <app-custom-input
+            [Identifier]="'kata-name'"
+            [Height]="'48px'"
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/editor-question/editor-question.component.html:55]: do not use magic numbers. create field for this and use it

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Comment by Limbo2332** [comment]
https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/93576084/66d11a30-3e3a-4537-835e-c2fa904fee95

**Review by apofis96 — APPROVED** [review]

**Pull request merged**